libraries/Microsoft.Bot.Builder.Ai/QnAMaker.cs

```diff
@@ -89,10 +89,13 @@ public QnAMaker(QnAMakerOptions options)
         {
             if (context.Request.Type == ActivityTypes.Message)
             {
-                var results = await this.GetAnswers(context.Request.AsMessageActivity().Text.Trim()).ConfigureAwait(false);
-                if (results.Any())
+                if (!string.IsNullOrEmpty(context.Request.AsMessageActivity().Text))
```

Comment:
Thank you for the PR. One quick comment:
Rather than going through the cast + check, can you update this to pull the .Text out to a variable, check that, and use it if it's relevant? 

The goal here that you're pushing towards is good.

Reply:
sure np, makes sense.